pkg/yurtctl/cmd/revert/edgenode.go

```diff
@@ -315,8 +303,8 @@ func (r *RevertEdgeNodeOptions) getKubeletSvcBackup() string {
 	return fmt.Sprintf(enutil.KubeletSvcBackup, r.KubeadmConfPath)
 }
 
-func (r *RevertEdgeNodeOptions) getYurthubYaml() string {
-	return filepath.Join(r.PodMainfestPath, enutil.YurthubYamlName)
+func (r *RevertEdgeNodeOptions) getYurthubYaml(podManifestPath string) string {
```

Comment:
How about remove func `getYurthubYaml()` as this func is not related to `RevertEdgeNodeOptions`?

Reply:
same as `ConvertEdgeNodeOptions`

Reply:
done